db/citations.yml

```diff
@@ -10151,15 +10163,37 @@ thatc;arxiv21:
   entrytype: article
   facility: nslsii, xpd, desy, P02.1
   grant: efrc18
-  journal: arXiv:2112.13300 [cond-mat]
+  journal: arxiv
   month: dec
   nb: ''
-  note: ''
+  note: arXiv:2112.13300 [cond-mat]
   number: ''
   optannote: ''
   optwwwlanl: ''
   pages: '2112.13300'
-  title: 'nmfMapping: a cloud-based web application for non-negative matrix factorization
+  professional_summary: Non negative matrix factorization (NMF) has been shown previously
+    to rapidly decompose large powder diffraction and atomic pair distribution function
+    (PDF) datasets into their constituent structural signals. Despite the utility
+    of NMF for quickly providing insight on large datasets, it is yet to gain widespread
+    adoption in the diffraction community. This is arguably due to a lack of a graphical
+    user interface or well written instructions to perform the analysis on diffraction
+    data. nmfMapping is presented in this paper as a web application that provides
+    an intuitive interface for exactly this purpose. This paper describes the interface
+    in detail, and provides two real examples of how the application has been used
+    to provide rapid insight into large in situ and operando datasets.
+  public_summary: Non negative matrix factorization can accelerate analysis of large
```

Comment:
for the public make it relevant to people's lives.    So something like "there is a need to find new materials for advanced applications such as batteries for electric vehicles.  Such materials are complex....." 

No jargon (in situ, operando, combinatorial diffraction...does a non-technical member of your family know or care what those are?)

Reply:
updated for review